lib/rules/no-restricted-imports.js

```diff
@@ -107,40 +116,68 @@ module.exports = {
             return memo;
         }, {});
 
-        // if no imports are restricted we don"t need to check
-        if (Object.keys(restrictedPaths).length === 0 && restrictedPatterns.length === 0) {
-            return {};
-        }
-
         const restrictedPatternsMatcher = ignore().add(restrictedPatterns);
 
-        /**
-         * Checks to see if "*" is being used to import everything.
-         * @param {Set.<string>} importNames Set of import names that are being imported
-         * @returns {boolean} whether everything is imported or not
-         */
-        function isEverythingImported(importNames) {
-            return importNames.has("*");
-        }
-
         /**
          * Report a restricted path.
+         * @param {string} importSource path of the import
+         * @param {Map<string,Object>} importNames Map of import names that are being imported
```

Comment:
A minor detail, the map has arrays of objects instead of objects now.

Reply:
Good catch 👍